R/bundle.R

```diff
@@ -485,7 +502,11 @@ addPackratSnapshot <- function(bundleDir, implicit_dependencies = c()) {
 
   # generate the packrat snapshot
   tryCatch({
-    performPackratSnapshot(bundleDir)
+    if (!file.exists(snapshotLockFile(bundleDir))) {
+      performPackratSnapshot(bundleDir)
+    } else {
+      message("Re-using Packrat Lock File")
```

Comment:
I think this is a pretty easy addition `capture.output(packrat::status())` would do it. But what should the default behavior be - try to use the lockfile and flag a warning, or fall back on performPackratSnapshot?